--- /etc/xdg/tint2/tint2rc	2016-08-02 15:44:51.000000000 +0200
+++ tint2rc	2017-03-29 09:46:58.581058206 +0200
@@ -55,7 +55,7 @@
 
 #-------------------------------------
 # Panel
-panel_items = LTSC
+panel_items = TSC
 panel_size = 100% 30
 panel_margin = 0 0
 panel_padding = 2 0 2
@@ -84,7 +84,7 @@
 taskbar_padding = 0 0 2
 taskbar_background_id = 0
 taskbar_active_background_id = 0
-taskbar_name = 1
+taskbar_name = 0
 taskbar_hide_inactive_tasks = 0
 taskbar_hide_different_monitor = 0
 taskbar_always_show_all_desktop_tasks = 0
@@ -145,8 +145,8 @@
 
 #-------------------------------------
 # Clock
-time1_format = %H:%M
-time2_format = %A %d %B
+time1_format = %H:%M %Z
+time2_format = %Y-%m-%d
 time1_timezone = 
 time2_timezone = 
 clock_font_color = #ffffff 100
